src/components/elements/inputs/dropdown.js

```diff
@@ -30,4 +26,8 @@ export default class DropdownList extends Input {
         value={this.state.value} />
     )
   }
+
+  isMultiSelect(){
```

Comment:
Make this a property too, and maybe move it into ```PrimitiveProperty``` since we are using it in both components?